modules/plot/main/coffee/charts/PieChart.coffee

```diff
@@ -18,6 +18,8 @@ class PieChart extends hx.EventEmitter
 
   constructor: (@selector, options) ->
     super
+    hx.select @selector
+      .on 'resize', 'hx.plot', => @render()
```

Comment:
This is where we set up the resize listener, so that when a detached div is added to the page the pie chart renders correctly.